.env.example

```diff
@@ -98,6 +98,7 @@ DC_ID=song.collab
 DC_URL=https://song.rdpc.cancercollaboratory.org/
 DC_BATCH_SIZE=50
 DC_GATEWAY_URL=https://api.rdpc-qa.cancercollaboratory.org/graphql
+# DC_ANALYSES_PAGESIZE=50
```

Comment:
commented out, meant as a "this is default"? or...

Reply:
Yes, as a general pattern in env schemas/example files i comment out the optional config parameters.

... I do now see here an old param I need to remove though, so, helpful comment to leave.

Reply:
had never seen that pattern, I like it, but it's not immediately intuitive. perhaps grouping them with a commented title "optional" would carry it the last ten yards

Reply:
This file is a bit of a mess in general, I don't want to get too carried away organizing it. There are other optional configs that this pattern could be applied to as well.